### PR TITLE
Update bookdash-selection.md

### PR DESCRIPTION
**Pull request by aleesteele**
### Summary

Discussing with @EstherPlomp of the @the-turing-way/book-dash-working-group - we thought this edit should be added to describe the responsibilities of being a part of the Book Dash Working group in a more binding way.

### What should a reviewer concentrate their feedback on?

- [ ] Everything looks ok?

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/the-turing-way/the-turing-way/blob/main/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/the-turing-way/the-turing-way/blob/main/README.md#contributors) in the README file:

### CODE REVIEW

File: book/website/community-handbook/bookdash/bookdash-selection.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ This subchapter explains the process of selection and logistics for organising t
 To ensure that the Book Dash continues to evolve as per the community needs and preferences, invite returning members of the Book Dash events to join a planning committee.
 Members of the Book Dash planning committee participate in the planning meetings, provide feedback on the ongoing process and when possible take the lead on organisation tasks related to the Book Dash.
 
-These tasks usually include the following roles and responsibilities:
+These tasks include completing one or more of the following roles and responsibilities:
 - **Planning Meeting**: attend a planning meeting to help select a date, agree on the format and share any feedback on the previous event that can help plan the Book Dash (time commitment: up to 30 minutes)
```
Comment (EstherPlomp) [inline book/website/community-handbook/bookdash/bookdash-selection.md:12]:
```suggestion
- **Planning Meetings**: attend a planning meeting to help select a date, agree on the format and share any feedback on the previous event that can help plan the Book Dash (time commitment: up to 30-60 minutes in the months leading to the Book Dash)
```

File: book/website/community-handbook/bookdash/bookdash-selection.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ This subchapter explains the process of selection and logistics for organising t
...
 - **Application Review**: review 2-3 applications based on your availability (time commitment: up to 30 minutes)
 - **Selection meeting**: participate in the selection meeting to help ensure that the selection is equitable (time commitment: 1.5-2 hours)
```
Comment (EstherPlomp) [inline book/website/community-handbook/bookdash/bookdash-selection.md:14]:
```suggestion
- **Selection meeting**: participate in the selection meeting to help ensure that the selection is equitable (time commitment: 60 minutes)
```

**Review by EstherPlomp — CHANGES_REQUESTED** [review]
Thanks Anne! I had a look and I think we need a couple more changes....

**Comment by aleesteele** [comment]
Good catch, thanks @EstherPlomp! 🙏

**Review by EstherPlomp — APPROVED** [review]
Thanks Anne for the updates!

**Comment by aleesteele** [comment]
Great, thanks @EstherPlomp !

**Pull request merged**